### DISCUSSION TITLE
Are pullRequestContributions limited?

### DISCUSSION THREAD

**Discussion by amannn** [community/community#24846] [Apps, API and Webhooks]
<p>I’m trying to compile a list of all public repositories that I’ve contributed via a pull request to. However it seems like it’s only possible to find nodes which aren’t older than a year – even when I use the cursor pagination.</p>
<p>Here’s the query I’m using:</p>
<pre><code>query ($after: String) {
  viewer {
    login
    contributionsCollection {
      pullRequestContributions(first: 100, after: $after) {
        totalCount
        edges {
          cursor
          node {
            pullRequest {
              id
              title
              url
              state
              baseRepository {
                isPrivate
                name
                url
                description
                owner {
                  login
                }
              }
            }
          }
        }
      }
    }
  }
}
</code></pre>
<p>Thanks!</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Yes, the <a href="https://developer.github.com/v4/object/user/#contributionscollection" rel="nofollow noopener"><code>contributionsCollection</code> field</a> defaults to a maximum of a year old if you don’t specify a different date range. See the <code>from</code> parameter documentation.</p>
<p>Let us know if you have more questions.</p>
